package.json

```diff
@@ -40,7 +40,8 @@
     "mustache": "^2.2.1",
     "mutationobserver-shim": "^0.3.1",
     "pbf": "^1.3.2",
-    "point-in-polygon": "1.0.0",
+    "point-geometry": "^0.0.0",
+    "point-in-polygon": "steve9164/point-in-polygon",
```

Comment:
I'll move the `point-in-polygon` repo over. `point-geometry` is a Mapbox module already used by `vector-tile`, and the output of some of the functions use the `Point` type from `point-geometry`.